### ISSUE TITLE
Batch consumer Exception

### ISSUE THREAD

**Issue by rnkoaa** [micronaut-projects/micronaut-kafka#676]
### Issue description

Hello I ran into Issue [#502](https://github.com/micronaut-projects/micronaut-kafka/issues/502) fixed and merged with PR  [#598](https://github.com/micronaut-projects/micronaut-kafka/pull/598). 

However, After attempting with latest versions, I'm still having deserialization exception.

I see that the PR was merged but it does not appear to be in a released version, Is there a timeline when a release with that PR will be cut?

Micronaut Version: 3.9.0-SNAPSHOT
Micronaut Kafka Version: 4.5.2

Below is a simple application to demonstrate the issue. 

[micronaut-simple-batch-consumer](https://github.com/rnkoaa/micronaut-simple-batch-consumer)

**Comment by guillermocalvo** [comment]
Hi @rnkoaa 

Thank you for reporting the issue. As you point out, this scenario is already solved by enhancement #502.

However, you don't strictly need it to make your sample app work. You could [configure your consumer's deserializers](https://micronaut-projects.github.io/micronaut-kafka/latest/guide/#_kafkalistener_and_deserializers) explicitly:

  1. You can do that via application config:
     - `kafka.consumers.simple-change-request-consumer-007.key.deserializer`
     - `kafka.consumers.simple-change-request-consumer-007.value.deserializer`
  2. Or you can define additional properties directly in `@KafkaListener`:
     ```java
     @KafkaListener(
       batch = true,
       properties = {
         @Property(name = "key.deserializer", value = "org.apache.kafka.common.serialization.StringDeserializer"),
         @Property(name = "value.deserializer", value = "org.apache.kafka.common.serialization.ByteArrayDeserializer")
       },
       offsetReset = OffsetReset.EARLIEST,
       groupId = "simple-change-request-consumer-007"
     )
     ```

In any case, you're going to want to configure your consumer with `StringDeserializer` and `ByteArrayDeserializer` to match its method definition:

```java
@Topic("change-requests-v1")
public void receive(List<ConsumerRecord<String, byte[]>> consumerRecords) {
  ...
}
```

If the method received `List<ConsumerRecord<String, String>>` instead, you would need to configure both deserializers as `StringDeserializer` to match the types of the consumer record.

As for the release timeline of enhancement #502, I will let @sdelamo speak to that.

For the moment, I'm closing this issue as a duplicate of issue #811

**Comment by rnkoaa** [comment]
thanks @guillermocalvo
